src/include/btmem.h

```diff
@@ -820,16 +827,14 @@ struct __wt_ref {
 #undef	ref_ikey
 #define	ref_ikey	key.ikey
 
-	union {
-		WT_PAGE_DELETED	*page_del;	/* Deleted page information */
-		WT_PAGE_LOOKASIDE *page_las;	/* Lookaside information */
-	};
+	WT_PAGE_DELETED	  *page_del;	/* Deleted page information */
+	WT_PAGE_LOOKASIDE *page_las;	/* Lookaside information */
 };
```

Comment:
As noted, it isn't great that `WT_REF` has grown, but I think this change or something very similar is going to be necessary for WT-3805, so I can live with it.